src/routes/view-content/[guideId]/[bibleSection]/feedback/BibleWellFeedback.svelte

```diff
@@ -69,12 +69,7 @@
                     likes,
                     improvements,
                 },
-                additionalProperties: {
-                    ...additionalProperties,
-                },
-                browserAndScreenSize: {
```

Comment:
The shape of the screen size data here will technically be a breaking change but since we don't have any feedback in prod it won't really matter in terms of data consistency.